### DISCUSSION TITLE
can't push, pull or clone, can fetch

### DISCUSSION THREAD

**Discussion by TomaszKozera** [community/community#141636] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I've been using Github repo in a project for past year, everything was working fine. I'm normally using SourceTree, repo uses LFS, I'm not using any VPN.

Last Thursday, during the day, I could no loner Push to repo - nothing was happening. I have Pushed maybe 30 minutes earlier with no issue. Trying it via command line it showed there is an issue with access rights.
I created a new SSH key, set everything up. It was still not working but there was no info regarding access rights.
![image](https://github.com/user-attachments/assets/961e7571-8ff0-413d-a164-6d5e0b958fb8)

I cleaned the repo:
```
git reset --hard
git clean -fd
```
Restarted PC, made a new PR from scratch - same thing.
I was able to fetch and pull though.

I checked that the key is working correctly:
`ssh -T [git@github.com](mailto:git@github.com)`

Reinstalling Git did not help, also tried `git init` and `push --force`.

Yesterday I was no longer able to Pull, could Fetch though.
I deleted whole repo folder, tried to Clone it again - didn't work. I uninstalled Git, LFS, rebooted, reinstalled git, LFS, rebooted. Now I can't clone the repo still.
![image](https://github.com/user-attachments/assets/5567375f-3864-4bb6-82fb-1f9a657f49bf)
I tried an even more verbose command:
`GIT_TRACE=1 GIT_TRACE_PACKET=1 git -c filter.lfs.smudge= -c filter.lfs.required=false -c diff.mnemonicprefix=false -c core.quotepath=false --no-optional-locks clone --progress --verbose --branch develop git@github.com:XXXX.git G:\YYYY\repo`
the output doesn't seem to give any more clues:
![image](https://github.com/user-attachments/assets/a570ff71-57de-4f92-9aba-8f2c2848078a)

**Comment by TomaszKozera** [comment, marked as answer]
A friend found a solution, yay!
https://stackoverflow.com/questions/11941175/git-fetch-pull-clone-hangs-on-receiving-objects/79075865#79075865

**Reply by AjayKodavati** [reply]
thanks for this!!!!

**Comment by Umeshjyani** [comment]
Hi everyone,

I’ve been using a GitHub repository for over a year without issues. I primarily work through SourceTree, the repo uses Git LFS, and I’m not using any VPN.

Problem:
Last Thursday, during the day, I suddenly couldn’t Push to the repository — nothing was happening. I had successfully pushed just 30 minutes earlier. When attempting to push via the command line, I received a message indicating an access rights issue.

What I’ve tried so far:

Created a new SSH key, added it to GitHub, and configured it locally.

Verified SSH key connection via:

**ssh -T git@github.com**

and it connected successfully.

Cleaned the repo using:

**git reset --hard**
**git clean -fd**


Restarted my PC.

Created a new local repo and tried pushing again — same issue.

Reinstalled Git and Git LFS, rebooted, and reattempted.

Tried git init and git push --force.

I was still able to Fetch and Pull until yesterday — now Pull fails too, though Fetch still works.

Current Situation:
After deleting the local repo folder entirely and attempting to clone the repo again, it still fails. I tried a verbose command to gather more details:

**GIT_TRACE=1 GIT_TRACE_PACKET=1 git -c filter.lfs.smudge= -c filter.lfs.required=false -c diff.mnemonicprefix=false -c core.quotepath=false --no-optional-locks clone --progress --verbose --branch develop git@github.com:XXXX.git G:\YYYY\repo**

But the output didn’t reveal any obvious errors or additional clues.

Notes:

SSH key is active and confirmed working.

No VPN or proxy in use.

No branch protection rules affecting pushes.

Repo uses Git LFS.

No recent changes on my local machine or network that I’m aware of.

Question:
Has anyone experienced a similar issue, or could suggest possible causes or debugging steps I might have missed?
Could this be a GitHub-side issue (e.g., SSH/IP restrictions, repo settings) or something environmental on my end?

Any help would be greatly appreciated.
